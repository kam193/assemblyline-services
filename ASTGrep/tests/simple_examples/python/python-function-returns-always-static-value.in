def I1I1I1lI(l11lIllI):
    return True

I1I1I1lI(123)

def I2(var):
    return f"{var}"

I2(123)