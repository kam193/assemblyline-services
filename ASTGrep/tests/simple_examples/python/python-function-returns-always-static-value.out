def I1I1I1lI(l11lIllI):
    return True

True

def I2(var):
    return f"{var}"

I2(123)